frontend/src/views/Collection/components/ActionButtons/index.tsx

```diff
@@ -1,63 +1,31 @@
-import { Button, Intent } from "@blueprintjs/core";
```

Comment:
This component used to live at the bottom of the collection page. But now it's refactored to include all the action buttons, such as Publish, Add, and MoreDropdown (Delete), and moved to the upper right corner

OLD:
<img width="1449" alt="Screen Shot 2021-03-31 at 12 31 32 PM" src="https://user-images.githubusercontent.com/6309723/113200354-104b2880-921d-11eb-99ed-523e67b6c686.png">

NEW:
<img width="1439" alt="Screen Shot 2021-03-31 at 12 33 17 PM" src="https://user-images.githubusercontent.com/6309723/113200548-4be5f280-921d-11eb-8efe-d49915aec1ba.png">